idiokit/dns/tests/test_dns_conf.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+import tempfile
+
+from .. import _conf
+
+
+class HostsFileTests(unittest.TestCase):
+    _missing_file = None
+    _empty_file = None
+    _hosts = None
+
+    def setUp(self):
+        missing = tempfile.NamedTemporaryFile()
+        self._missing_file = missing.name
+        missing.close()  # Close deletes the file
+
+        self._empty_file = tempfile.NamedTemporaryFile()
+        self._empty_file.flush()
+
+        # Make sure empty and missing files are not same
+        self.assertNotEqual(self._empty_file.name, self._missing_file)
```

Comment:
Could you get rid of this assertion just by calling ```missing.close()``` after you have created self._empty_file?